data/data/ibmcloud/variables-ibmcloud.tf

```diff
@@ -70,7 +75,7 @@ variable "ibmcloud_vpc_permitted" {
 variable "ibmcloud_vpc" {
   type        = string
   description = "The name of an existing cluster VPC."
-  default     = null
+  default     = ""
```

Comment:
Is this change related to this PR?

Reply:
No, I was cleaning up the reference as I found `null` isn't desired for string.